### PR TITLE
didyoumean: 1.1.0 -> 1.1.3

### LINKED ISSUE #184088: didyoumean: 1.1.0 -> 1.1.3
###### Checklist

- [X] Checked the [nixpkgs master branch](https://github.com/NixOS/nixpkgs)
- [X] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

###### Project name
`nix search` name: didyoumean

current version: 1.1.0
desired version: 1.1.3

###### Notify maintainers

maintainers: @evanjs 

###### Note for maintainers

Version 1.1.3 comes with a man page as well as shell completions for a few shells (bash, zsh, fish, elvish). I'd like to have the script install these. I'm more than willing to do the actual update myself, but I'm very new to Nix, so I'll probably need some guidance, specifically on how to install these completions and the man page, and how to update the `sha256` hash.

Thanks!

### PR DESCRIPTION
**Pull request by wegank**
###### Description of changes

https://github.com/hisbaan/didyoumean/releases/tag/v1.1.3

Closes #184088.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by viraptor** [comment]
Result of `nixpkgs-review pr 186557` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>didyoumean</li>
  </ul>
</details>

File: pkgs/tools/misc/didyoumean/default.nix [hunk 1]
```diff
@@ -2,32 +2,54 @@
...
+  ];
+
+  postInstall = ''
+    installManPage man/dym.1
+    installShellCompletion completions/dym.{bash,fish}
+    installShellCompletion --zsh completions/_dym
+  '';
+
+  # Clipboard doesn't exist in test environment
+  doCheck = false;
```
Comment (Mindavi) [inline pkgs/tools/misc/didyoumean/default.nix:46]: Are _all_ tests broken? Always nice to have some working tests.

Comment (wegank) [reply pkgs/tools/misc/didyoumean/default.nix:46]: There is only _one_ test in the project.

Comment (Mindavi) [reply pkgs/tools/misc/didyoumean/default.nix:46]: Ah, too bad then

**Pull request merged**